docker-compose.yml

```diff
@@ -7,47 +7,54 @@ services:
                 - SECRET_KEY=ineedtoputasecrethere
                 - DATABASE_URL=sqlite:///databasegateway.db
         network_mode: "host"
+    redis:
+        image: redis:latest
+        network_mode: "host"
     bob:
         image: openmined/grid-node:latest
         environment:
                 - GRID_NETWORK_URL=http://localhost:5000
                 - ID=Bob
                 - ADDRESS=http://localhost:3000/
-                - DATABASE_URL=sqlite:///databaseBob.db
+                - REDISCLOUD_URL=redis:///localhost:6379
```

Comment:
@IonesioJunior  isn´t this one '/' too much? Three were needed for the database url but for redis this should be two: `REDISCLOUD_URL=redis://localhost:6379`